modules/installation-about-restricted-network.adoc

```diff
@@ -12,6 +12,7 @@
 // * installing/installing_ibm_powervs/installing-restricted-networks-ibm-power-vs.adoc
 // * installing/installing-rhv-restricted-network.adoc
 // * installing/installing-restricted-networks-nutanix-installer-provisioned.adoc
+// * installing/installing-restricted-networks-azure-installer-provisioned.adoc
```

Comment:
Following lines should not be displayed for ipi method. It is specific for upi.
https://github.com/openshift/openshift-docs/pull/61086/files#diff-0995ccf569e68d8a5e211b07d6f6bd89d741f4761fb517528c10bf7b445f9593R67-R72